README.md

```diff
@@ -56,8 +56,12 @@ Running in Docker
 - `docker compose up --build --exit-code-from regtest` - To run regression tests in a Docker environment.
 
 Running in Kubernetes
-- `./setup.sh` - To run Polaris as a mini-deployment locally. This will create two pods that bind themselves to port `8181`.
-- `kubectl port-forward svc/polaris-service -n polaris 8181:8181` - To create a secure connection between a local machine and a pod within the cluster.
+- `./setup.sh` - To run Polaris as a mini-deployment locally. This will create one pod that bind itself to ports `8181` and `8182`.
+- `kubectl port-forward svc/polaris-service -n polaris 8181:8181` - To create a secure connection between a local machine and a pod within the cluster for service endpoints.
+- `kubectl port-forward svc/polaris-service -n polaris 8182:8182` - To create a secure connection between a local machine and a pod within the cluster for metrics endpoints.
```

Comment:
Can we combine these two commands to one like this?
```
kubectl port-forward svc/polaris-service -n polaris 8181:8181 8182:8182
```

Reply:
Took care.